preparedata/src/main/resources/org/orbisgis/orbisprocess/geoclimate/preparedata/bdtopo/importPreprocess.sql

```diff
@@ -135,6 +146,24 @@ CREATE TABLE $INPUT_VEGET AS SELECT a.the_geom, a.ID as ID_SOURCE, a.NATURE as T
 -- Update the VEGET table with the new appropriate type key, coming from the abstract table
 UPDATE $INPUT_VEGET SET TYPE=(SELECT c.TERM FROM $VEGET_BD_TOPO_TYPE b, $VEGET_ABSTRACT_TYPE c WHERE c.ID_TYPE=b.ID_TYPE and ${INPUT_VEGET}.TYPE=b.NATURE) WHERE TYPE IN (SELECT b.NATURE FROM $VEGET_BD_TOPO_TYPE b);
 
+-------------------------------------
+-- Impervious areas (from the layers "TERRAIN_SPORT", "CONSTRUCTION_SURFACIQUE", "SURFACE_ROUTE" and "SURFACE_ACTIVITE") that are in the study area (ZONE)
+-------------------------------------
+DROP TABLE IF EXISTS $TMP_IMPERV_TERRAIN_SPORT, $TMP_IMPERV_CONSTRUCTION_SURFACIQUE, $TMP_IMPERV_SURFACE_ROUTE, $TMP_IMPERV_SURFACE_ACTIVITE;
+CREATE TABLE $TMP_IMPERV_TERRAIN_SPORT AS SELECT a.the_geom, a.ID as ID_SOURCE FROM $TERRAIN_SPORT a, $ZONE b WHERE a.the_geom && b.the_geom AND ST_INTERSECTS(a.the_geom, b.the_geom) AND (a.NATURE='Piste de sport' OR a.NATURE='Terrain de tennis');
```

Comment:
Have a look to the [def](http://professionnels.ign.fr/doc/DC_BDTOPO_2.pdf#page=104).
Seems to be only the track, even if there is some errors in their def since Kercado track is noted as 'Piste de sport' even if they wrote 'Les pistes d'athlétisme sont exclues. '

Reply:
funny when I think about "pistes de sport" I had in mind specifically athetism track whereas they are not included... 
OK to keep them. Why do you want to get rid of tennis court ? Because it may be inside courts (it is not written) ?

Reply:
I overlayed them with OSM and seems there is many green / grass / clay areas.

Reply:
joker The bdtopo v2 lacks the information to qualify this type of feature.I propose not to keep them

Reply:
Tennis court have been removed. We only keep sport track